### PR TITLE
Update apache-sshd.md

### PR DESCRIPTION
**Pull request by FliegenKLATSCH**

### CODE REVIEW

File: _impls/apache-sshd.md [hunk 1]
```diff
@@ -6,49 +6,66 @@ license: "[Apache-2.0](http://www.apache.org/licenses/LICENSE-2.0)"
...
 client: yes
 server: yes
 library: both
 protocols:
     cipher:
-        - aes128-ctr
-        - aes192-ctr
-        - aes256-ctr
-        - arcfour256
+        - aes128ctr
```
Comment (fingolfin) [inline _impls/apache-sshd.md]: That's wrong, I am afraid. These strings must match *exactly* what the RFC mandates.

See also https://github.com/apache/mina-sshd/blob/808cdeaccaeb9e42670c06cae175dafa0e9ce21a/sshd-common/src/main/java/org/apache/sshd/common/cipher/BuiltinCiphers.java#L320-L329

File: _impls/apache-sshd.md [hunk 2]
```diff
@@ -6,49 +6,66 @@ license: "[Apache-2.0](http://www.apache.org/licenses/LICENSE-2.0)"
...
         - zlib
         - zlib@openssh.com
         - none
     hostkey:
-        - ssh-rsa
         - ssh-dss
-        - ecdsa-sha2-nistp256
-        - ecdsa-sha2-nistp384
-        - ecdsa-sha2-nistp521
+        - ssh-rsa
```
Comment (fingolfin) [inline _impls/apache-sshd.md:32]: Please don't reorder without need, that just makes the diff bigger. I mean, I would understand if you e.g. sorted everything alphabetically (but if you feel tempted to do so, *please* do it in a separate commit); but this seems just random, as `ssh-rsa` now is followed by `rsa-sha2` etc.

File: _impls/apache-sshd.md [hunk 3]
```diff
@@ -6,49 +6,66 @@ license: "[Apache-2.0](http://www.apache.org/licenses/LICENSE-2.0)"
...
+        - rsa-sha2-256
+        - rsa-sha2-512
+        - nistp256
+        - nistp384
+        - nistp521
```
Comment (fingolfin) [inline _impls/apache-sshd.md]: These are not valid hostkey formats.

File: _impls/apache-sshd.md [hunk 4]
```diff
@@ -6,49 +6,66 @@ license: "[Apache-2.0](http://www.apache.org/licenses/LICENSE-2.0)"
...
     compression:
         - zlib
         - zlib@openssh.com
         - none
     hostkey:
-        - ssh-rsa
         - ssh-dss
-        - ecdsa-sha2-nistp256
-        - ecdsa-sha2-nistp384
-        - ecdsa-sha2-nistp521
```
Comment (fingolfin) [inline _impls/apache-sshd.md:36]: Are you sure those were removed? There are still tests for them in the mina-sshd repository

File: _impls/apache-sshd.md [hunk 5]
```diff
@@ -6,49 +6,66 @@ license: "[Apache-2.0](http://www.apache.org/licenses/LICENSE-2.0)"
...
+        - ecdsa-sha2-nistp256-cert-v01@openssh.com
+        - ecdsa-sha2-nistp384-cert-v01@openssh.com
+        - ecdsa-sha2-nistp521-cert-v01@openssh.com
     kex:
-        - ecdh-sha2-nistp521
-        - ecdh-sha2-nistp384
-        - ecdh-sha2-nistp256
+        - diffie-hellman-group1-sha1
         - diffie-hellman-group-exchange-sha256
-        - diffie-hellman-group-exchange-sha1
```
Comment (fingolfin) [inline _impls/apache-sshd.md:42]: contradicts https://github.com/apache/mina-sshd/blob/0da9e23c693323fe1d4f42ee702f0c2d8a1bd534/sshd-core/src/main/java/org/apache/sshd/common/kex/BuiltinDHFactories.java#L400

File: _impls/apache-sshd.md [hunk 6]
```diff
@@ -6,49 +6,66 @@ license: "[Apache-2.0](http://www.apache.org/licenses/LICENSE-2.0)"
...
     mac:
-        - hmac-md5
-        - hmac-sha1
-        - hmac-sha2-256
-        - hmac-sha2-512             # was brokeb before 1.1.0
-        - hmac-sha1-96
-        - hmac-md5-96
+        - hmacmd5
+        - hmacmd596
+        - hmacsha1
```
Comment (fingolfin) [inline _impls/apache-sshd.md]: These are not valid MAC names

File: _impls/apache-sshd.md [hunk 7]
```diff
@@ -6,49 +6,66 @@ license: "[Apache-2.0](http://www.apache.org/licenses/LICENSE-2.0)"
...
+        - diffie-hellman-group17-sha512
+        - diffie-hellman-group18-sha512 
+        - ecdh-sha2-nistp256
+        - ecdh-sha2-nistp384
+        - ecdh-sha2-nistp521
     mac:
-        - hmac-md5
-        - hmac-sha1
-        - hmac-sha2-256
-        - hmac-sha2-512             # was brokeb before 1.1.0
```
Comment (fingolfin) [inline _impls/apache-sshd.md:49]: Why was the comment removed?

**Review by fingolfin — CHANGES_REQUESTED** [review]
Thank you! However, I have some concerns about this PR.

**Comment by FliegenKLATSCH** [comment]
I just copy pasted from their readme, was apparently not the best idea :$

**Review by fingolfin — APPROVED** [review]

**Pull request merged**